ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-main.yaml

```diff
@@ -74,6 +74,11 @@ tests:
   steps:
     cluster_profile: aws-2
     workflow: openshift-upgrade-aws
+- as: e2e-aws-ovn-upgrade-check-dev-symlinks
+  cron: '@daily'
```

Comment:
Sorry for the late dropin but the preference for periodics is to use a release specific [__periodics.yaml](https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/#configuration-for-periodic-jobs) variant config so that tooling like Sippy and Component Readiness can determine the release and ingest the jobs.

Reply:
We're in process of moving all our periodics into __periodics.yaml in [a separate PR](https://github.com/openshift/release/pull/66610), we will sync them depending on what PR merges first.